CHANGELOG.md

```diff
@@ -5,6 +5,18 @@
 * Breaking Changes
   * All SDK error enums are now internal
   * See [list of new / updated error cases and codes](SDK_ERROR_CODES.md)
+* BraintreeVenmo
```

Comment:
Can also move this grouping above the breaking changes section - this is all additions there is nothing that is considered a breaking change in this PR